arbitrum-docs/for-devs/third-party-docs/Particle/particle.md

```diff
@@ -0,0 +1,266 @@
+
+Upon calling `handleLogin`, the user will be taken through the defined social login mechanism, after which an EOA will be generated (using MPC-TSS) and assigned to the smart account configured previously.
+
+Your usage of `connect` may look something like this (although, as mentioned, it can be used in isolation if preferred).
+
+```typescript
+const handleLogin = async (authType) => {
+    if (!userInfo) {
+      await connect({
+        socialType: authType,
+        chain: ArbitrumSepolia
+      });
+    }
+};
+```
+
+### Executing a gasless transaction
+
+At this point, a user has logged in through their social account and been assigned a smart account. We'll need to build a simple function to test it, specifically through a gasless transaction.
+
+Because we're using Ethers, this can be a simple transaction construction and execution, using the `sendTransaction` method on a `signer` object from `{your provider object}.getSigner()`.
+
+This method will take a standard transaction object. In this case, we'll simply define an object containing:
+- `to`, the recipient of the transaction. We can set this to `0x000000000000000000000000000000000000dEaD` for the sake of the demo.
+- `value`, the amount of ETH you'd like to send to the recipient address.
+
+This object (we'll define it as `tx`) can then be passed into `signer.sendTransaction(tx)`. Upon calling, the user will be asked to confirm a transaction (sign a UserOperation hash) through an embedded popup within your application. After doing so, the transaction will be executed on-chain.
+
+Following the example described above, your function may look similar to the snippet shown below.
+
+```typescript
+const executeUserOp = async () => {
+    const signer = customProvider.getSigner();
+
+    const tx = {
+      to: "0x000000000000000000000000000000000000dEaD",
+      value: ethers.utils.parseEther("0.001"),
+    };
+
+    const txResponse = await signer.sendTransaction(tx);
+    const txReceipt = await txResponse.wait();
+
+    notification.success({
+      message: "Transaction Successful",
+      description: (
+        <div>
+          Transaction Hash: <a href={`https://sepolia.arbiscan.io/tx/${txReceipt.transactionHash}`} target="_blank" rel="noopener noreferrer">{txReceipt.transactionHash}</a>
+        </div>
+      )
+    });
+  };
+```
+
+This transaction will be gasless because we're meeting two conditions The first is whether or not we've expressed that this transaction should be sponsored (done earlier by including `SendTransactionMode.Gasless` within `AAWrapProvider`). The second condition is the existence of adequate funds to sponsor the transaction. Because we're on a Testnet (Arbitrum Sepolia), all transactions are automatically sponsored without the need to deposit USDT. However, if this were on Arbitrum One or Arbitrum Nova, the Paymaster shown on the [Particle dashboard](https://dashboard.particle.network) would need to be funded.
```

Comment:
```suggestion
This transaction will be gasless because we're meeting two conditions. The first is whether or not we've expressed that this transaction should be sponsored (done earlier by including `SendTransactionMode.Gasless` within `AAWrapProvider`). The second condition is the existence of adequate funds to sponsor the transaction. Because we're on a Testnet (Arbitrum Sepolia), all transactions are automatically sponsored without the need to deposit USDT. However, if this were on Arbitrum One or Arbitrum Nova, the Paymaster shown on the [Particle dashboard](https://dashboard.particle.network) would need to be funded.
```